Unify domain colors in ComplexBlocks

christiankral opened the pull request:
close #2833

beutlich commented:
This commit introduces large scale changes and I cannot judge if they are introduced intentionally or if they are noise.

christiankral commented:
> This commit introduces large scale changes and I cannot judge if they are introduced intentionally or if they are noise.

I changed everything in a few commits.

- I changed the colors to ComplexBlock blue
- I removed the unnecessary line art from the diagram view, where not required, according to the guidelines in https://doc.modelica.org/Modelica%203.2.3/Resources/helpOM/Modelica.UsersGuide.Conventions.Icons.html

I made no changes to the code format intentionally. Most noise is generated by Dymola automatically -- by a reason unknown to me. I checked the entire MSL and and the check is OK. If you checkout the branch and visually go through the models you will get a good impression of the changes.

christiankral commented:
OK, I didn't notice.
